config.ini

```diff
@@ -0,0 +1,20 @@
+[Darwin]
+; MATLAB_BIN=Applications/MATLAB_R2014a.app/bin
```

Comment:
May as well use `#` for lines that are commented out. I find this special syntax to be just confusing.